robusta_krr/core/models/config.py

```diff
@@ -103,7 +103,9 @@ def validate_resources(cls, v: Union[list[str], Literal["*"]]) -> Union[list[str
         if v == []:
             return "*"
 
-        return [val.capitalize() for val in v]
+        # NOTE: KindLiteral.__args__ is a tuple of all possible values of KindLiteral
+        # So this will preserve the big and small letters of the resource
+        return [next(r for r in KindLiteral.__args__ if r.lower() == val.lower()) for val in v]
```

Comment:
Just about to send my patch and got conflicted w/ this PR.  😅

JFYR.
You may consider to use `typing.get_args()` instead of accessing `__args__` directly next time.